knowledge_repo/app/templates/base.html

```diff
@@ -104,18 +106,6 @@
             </li>
           </ul>
 
-          <!-- Expose the same links as above in the collapsed navbar header -->
```

Comment:
what is this for?!

Reply:
This is for the responsive menus. You don't want to delete it :).

Reply:
kk - I'll have to ask you about that later but putting this back